src/doc/trpl/guessing-game.md

```diff
@@ -239,7 +239,7 @@ use std::rand;
 fn main() {
     println!("Guess the number!");
 
-    let secret_number = (rand::random::<uint>() % 100u) + 1u;
+    let secret_number = (rand::random::<u32>() % 100u) + 1u;
```

Comment:
So the best way to go about this is to just make the additional changes to this file, and then:
```
git add guessing-game.md
git commit --amend
(do the commit)
git push -f origin master
```
That will update your PR with the new changes!